MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: post
+title: Validation in .NET MAUI DataForm control | Syncfusion
```

Comment:
title:  Data validation in .NET MAUI DataForm control

Reply:
change title as stated by jeyasri

Reply:
Changed title